sdks/java/core/pom.xml

```diff
@@ -315,6 +315,19 @@
       <scope>test</scope>
     </dependency>
 
+    <!-- 
+    bytebuddy has provided scope dep on SuppressFBWarnings 
+    annotations (to avoid a real dep) and these are
+    retained on certain .class files, necessitating
+    this provided dependency to instantiate those classes
+    -->
+    <dependency>
```

Comment:
Cancel that - we just won't test at this fine granularity. A change-detector test suffices.